esmvaltool/diag_scripts/__init__.py

```diff
@@ -1 +1 @@
-"""ESMValTool diagnostic scripts."""
+"""Initialize the ESMValTool thermodyn_diagtool package."""
```

Comment:
Please undo the changes to this file

Reply:
Done. Thank you.

Reply:
Did you commit and push your changes? Because I don't see it on GitHub

Reply:
@ValerioLembo This is still not resolved

Reply:
Sorry, I missed that...

Reply:
This is still not solved